src/pycolmap/scene/reconstruction.cc

```diff
@@ -64,6 +64,35 @@ void BindReconstruction(py::module& m) {
       .def("num_reg_images", &Reconstruction::NumRegImages)
       .def("num_points3D", &Reconstruction::NumPoints3D)
       .def("num_image_pairs", &Reconstruction::NumImagePairs)
+      .def(
+          "camera",
+          [](Reconstruction& self, camera_t camera_id) {
+            const Camera& camera = self.Camera(camera_id);
+            return camera;
+          },
+          "camera_id"_a)
+      .def(
+          "image",
+          [](Reconstruction& self, image_t image_id) {
+            const Image& image = self.Image(image_id);
+            return image;
+          },
+          "image_id"_a)
+      .def(
+          "point3D",
+          [](Reconstruction& self, point3D_t point3D_id) {
+            const Point3D& point3D = self.Point3D(point3D_id);
+            return point3D;
+          },
+          "point3D_id"_a)
+      .def(
+          "image_pair",
+          [](Reconstruction& self, image_pair_t pair_id) {
+            const Reconstruction::ImagePairStat& image_pair =
+                self.ImagePair(pair_id);
+            return image_pair;
+          },
+          "pair_id"_a)
```

Comment:
Is this needed? We can already do `reconstruction.images[image_id]`.

Reply:
Indeed. This is resolved. Thanks